packages/vscode-ui5-language-assistant/README.md

```diff
@@ -134,6 +134,17 @@ The feature is available in the following:
 
   - Enum values
   - SAPUI5 Namespaces
+  - any keys of property binding info
```

Comment:
Do we need "any" here? Please also start with the capital letter to follow the pattern above